tests/test_models.py

```diff
@@ -1098,6 +1098,16 @@ def test_annotate_deep_denoiser():
         assert annotations[i].data.shape == (3000,)
 
 
+def test_annotate_seisdae():
+    stream = obspy.read()
```

Comment:
Maybe it would be better to replace this with an artificially created random stream of fixed length. Then the function could also be parametrized to check that it also works if the input doesn't exactly have the length of the window size.